files/templates/userpage.html

```diff
@@ -104,7 +104,9 @@ <h1 class="font-weight-bolder h3 mb-0"><span {% if u.patron %}class="patron" sty
 				{% else %}<pre></pre>
 				{% endif %}
 				
-				<div class="font-weight-bolder mb-2"><a class="mr-1" href="/@{{u.username}}/upvoters">Upvoters</a> | <a class="mx-1" href="/@{{u.username}}/downvoters">Downvoters</a> | <a class="mx-1" href="/@{{u.username}}/upvoting">Upvoted</a> | <a class="ml-1" href="/@{{u.username}}/downvoting">Downvoted</a></div>
+				{% if u.admin_level >= 2 %}
```

Comment:
Here and in the other change below in this file, you're using `u`, which is the user who's page you are currently viewing. Pretty sure this ought to be `v`, the user you are currently logged in as - currently anyone can see the links only if the user they're viewing is admin.

Reply:
Dangit, you're correct. Fixed.